2-ui/99-ui-misc/01-mutation-observer/article.md

```diff
@@ -252,14 +252,14 @@ let mutationRecords = observer.takeRecords();
 ```
 
 ```smart header="Garbage collection interaction"
-Observers use weak references to nodes internally. That is: if a node is removed from DOM, and becomes unreachable, then it becomes garbage collected.
+Observers use weak references to nodes internally. That is: if a node is removed from the DOM, and becomes unreachable, then it can be garbage collected.
 
 The mere fact that a DOM node is observed doesn't prevent the garbage collection.
 ```
 
 ## Summary  
 
-`MutationObserver` can react on changes in DOM: attributes, added/removed elements, text content.
+`MutationObserver` can react to changes in DOM: attributes, added/removed elements, text content.
```

Comment:
```suggestion
`MutationObserver` can react to changes in DOM - attributes, add/remove elements, text content, etc.
```

Reply:
Hi, thanks for the help. I think this part of the article is referring to the Mutation Observer being able to detect added or removed elements.